### PR TITLE
Implement blog posts previews

### PR DESCRIPTION
**Pull request by noisersup**
# Description

Closes FerretDB/engineering#57.

## Readiness checklist

* [ ] I added/updated unit tests.
* [ ] I added/updated integration/compatibility tests.
* [ ] I added/updated comments and checked rendering.
* [ ] I made spot refactorings.
* [ ] I updated user documentation.
* [ ] I ran `task all`, and it passed.
* [x] I ensured that PR title is good enough for the changelog.
* [x] (for maintainers only) I set Reviewers ([`@FerretDB/core`](https://github.com/orgs/FerretDB/teams/core)), Assignee, Labels, Project and project's Sprint fields.
* [x] I marked all done items in this checklist.

### CODE REVIEW

File: .github/workflows/webpage-preview.yml [hunk 1]
```diff
@@ -23,7 +23,68 @@ env:
...
+  blog-preview:
+    name: Preview blog
+    runs-on: ubuntu-22.04
+    timeout-minutes: 10
+
+    if: >
+      contains(github.event.pull_request.labels.*.name, 'trust') &&
+      !contains(github.event.pull_request.labels.*.name, 'no ci') &&
+      (contains(github.event.pull_request.labels.*.name, 'blog/engineering') ||
+      contains(github.event.pull_request.labels.*.name, 'blog/marketing'))
```
Comment (noisersup) [inline .github/workflows/webpage-preview.yml:35]: As we have only 2 labels for blogs I've decided to not overcomplicate the check. In the future we might iterate through labels to check if they contain `blog/`

**Review by AlekSi — APPROVED** [review]

File: .github/workflows/webpage-preview.yml [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ---
-name: Docs Preview
+name: Webpage Preview
```
[the remark this answers is not in the document]
Comment (noisersup) [reply .github/workflows/webpage-preview.yml]: done

**Review by AlekSi — APPROVED** [review]

**Review by w84thesun — APPROVED** [review]
Nice!

**Review by rumyantseva — APPROVED** [review]
:shipit:

**Pull request merged**